src/partials/features-zigzag.tsx

```diff
@@ -100,17 +98,48 @@ function FeaturesZigZag() {
                 className="max-w-xl md:max-w-none md:w-full mx-auto md:col-span-5 lg:col-span-6 mb-8 md:mb-0 rtl"
                 data-aos="fade-up"
               >
-                <div className="max-w-full mx-auto md:max-w-none h-auto flex flex-row flex-wrap fill-[#1c1e21] dark:fill-[#e3e3e3]">
-                  <CloudbeesSvg className="flex-auto h-20 w-20 m-4" />
-                  <SplitSvg className="flex-auto h-20 w-20 m-4" />
-                  <FliptSvg className="flex-auto h-20 w-20 m-4" />
-                  <GoFeatureFlagSvg className="flex-auto h-20 w-20 m-4" />
-                  <FlagsmithSvg className="flex-auto h-20 w-20 m-4" />
-                  <LaunchDarklySvg className="flex-auto h-20 w-20 m-4" />
-                  <FlagdSvg className="flex-auto h-20 w-20 m-4" />
-                  <DevCycleSvg className="flex-auto h-20 w-20 m-4" />
-                  <UnleashSvg className="flex-auto h-20 w-20 m-4" />
-                  <HarnessSvg className="flex-auto h-20 w-20 m-4" />
+                <div
+                  className="max-w-full mx-auto md:max-w-none h-auto flex flex-row flex-wrap fill-[#1c1e21] dark:fill-[#e3e3e3]">
+                  <Link to="/ecosystem?instant_search[refinementList][vendor][0]=CloudBees"
```

Comment:
This link works but causes a redirect on the ecosystem page that messes with the history. Could you update the link to avoid this?

Reply:
encoded the URI and now the history works fine